src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationCard.vue

```diff
@@ -100,8 +100,7 @@ export default defineComponent({
   },
   methods: {
     handleStartNewApplication() {
-      this.applicationStore.upsertDraftApplication();
-      this.$router.push("/application");
+      this.$router.push("/application/certification");
```

Comment:
Small change, I've been using named routes whenever I can... I _think_ build will fail when we don't route to a name that exists in router... Whereas with path it is not caught, if we were to change the route location. 

Eg.
`this.$router.push({name: 'application-certification'});`

Reply:
oh interesting, yep changed to a named route instead.